sdk/src/frame.cpp

```diff
@@ -5,27 +5,17 @@ namespace aditof {
 
 Frame::Frame() : m_impl(new FrameImpl) {}
 
-Frame::~Frame() {
-    if (m_impl)
-        delete m_impl;
-}
+Frame::~Frame() = default;
 
-Frame::Frame(Frame &&other) noexcept {
-    this->m_impl = other.m_impl;
-    other.m_impl = nullptr;
-};
+Frame::Frame(Frame &&other) noexcept = default;
```

Comment:
Oare nu ar fi mai ok sa fie ca aici, fara nume (other) : System::System(System &&)

Reply:
Friendly reminder - Interaction on github should be in plain English please

Reply:
Modified to Frame::Frame(Frame &&). Thanks.